tests/hmm/plotting.py

```diff
@@ -366,15 +366,20 @@ def create_variated_histos(
                 hret = hdict[sname]
             ret[sname2] = hret
     return ret
-
 def create_datacard(dict_procs, parameter_name, all_processes, histname, baseline, variations, weight_xs):
     
     ret = Results(OrderedDict())
     event_counts = {}
-
     hists_mc = []
- 
-    #print("create_datacard processes=", all_processes)
+    proc_grps = [
```

Comment:
To make it simpler to manage the code in the future, could you load `proc_grps` from pars.py, so it's not hardcoded?

Reply:
done